src/common/constant.ts

```diff
@@ -1,20 +1,39 @@
 import { ElementType } from 'react';
 
-import { MainLayout } from '../components/Layout';
 import { Dashboard as DashboardComponent } from '../pages/Dashboard';
 import { Login as LoginComponent } from '../pages/Login';
 import { Register as RegisterComponent } from '../pages/Register';
+import { Games as GamesComponent } from '../pages/Games';
 import DashboardLayout from '../Layout/DashboardLayout';
 import GuestLayout from '../Layout/GuestLayout';
 
 
-export const enum AppPath {
-	Login = '/login',
-	Register = '/register',
-	LandingPage = '/landingPage',
-	Dashboard = '/dashboard',
+export const enum PageName {
+	LOGIN = 'Login',
+	REGISTER = 'Register',
+	LANDING_PAGE = 'LandingPage',
+	DASHBOARD = 'Dashboard',
+	GAMES = 'MyProject',
+	USERS = 'Users',
+	CLIENTS = 'Client',
+	PAYMENT = 'PaymentPlan'
 }
 
+export const convertNameToPath = (pageName: PageName) => {
+	const path = {
+		[PageName.LOGIN]: '/login',
```

Comment:
use global var to optimize performance:
const pathMap = new Map<string, string>(
[
[PageName.REGISTER,  '/register'],
[PageName.LANDING_PAGE: '/landingPage'],
...
]
)
export const convertNameToPath = (pageName: PageName) => pathMap.get(pageName)